backend/tests/skills/test_sandbox_attachment_tools.py

```
@@ -0,0 +1,131 @@
+    def mock_dependencies(self):
+        """Set up mock dependencies for provider import."""
+        with patch.dict(
+            "sys.modules",
+            {
+                "chat_shell.skills": MagicMock(),
+                "langchain_core.tools": MagicMock(),
+            },
+        ):
+            yield
+
+    def test_supported_tools_includes_attachment_tools(self, mock_dependencies):
```

Ruff (0.14.13)

Unused method argument: `mock_dependencies`

(ARG002)